go/vt/vtgate/evalengine/integration/comparison_test.go

```diff
@@ -681,3 +685,27 @@ func TestCaseExprWithValue(t *testing.T) {
 		}
 	}
 }
+
+func TestCeil(t *testing.T) {
+	var conn = mysqlconn(t)
+	defer conn.Close()
+
+	var ceilInputs = []string{
+		"0",
+		"1.5",
+		"-1.5",
+		"'1.5'",
+		"NULL",
+		"'ABC'",
+		"1.5e0",
+		"-1.5e0",
+		"9223372036854775810.4",
+		"7223372036854775010.1",
+		"2147483640.1",
+		"9223372036854775010.4",
+	}
```

Comment:
Good start with the tests. Can you think of more corner cases that would be worth adding here? Larger negative floats? Larger negative decimals?